test/lit/passes/trace-calls.wast

```diff
@@ -0,0 +1,151 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
+
+;; RUN: wasm-opt --enable-simd --trace-calls="noparamsnoresults,singleparamnoresults,multiparamsnoresults:tracempnr,noparamssingleresult,multiparamssingleresult" %s -S -o - | filecheck %s
```

Comment:
I don't seem to see an explanation for this notation yet? I suggested before either putting that in the cpp file or the `--help` output.

I think an explanation is needed because I can guess that `,` separates functions, but it's not obvious what `:` does...

Reply:
I've added explanation in the CPP file, I hope that's enough but let me know if you'd like this to be included in the help string as well